test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
Could use some help here.

[These lines](https://github.com/HabitRPG/habitrpg/blob/develop/test/api.mocha.coffee#L340-L349) should register three new users, and when they're all done, the `_party` variable should be an array of user objects, the newly registered ones. Instead, it's an array of 3 user objects, all the the same user, the one used to make the api calls.